SECURITY.md

```diff
@@ -1,11 +1,142 @@
+Send an email to [security@aptosfoundation.org](mailto:security@aptosfoundation.org) and provide your GitHub username. The team will create a new draft security advisory for further discussion.
 
-You can also report security bugs by emailing us directly at security@aptoslabs.com.
+For security reasons, DO NOT include attachments or provide detail sufficient for exploitation regarding the security issue in this email. Instead, wait for the advisory to be created, and **provide any sensitive details in the private GitHub advisory**.
 
-Thank you for improving the security of the Aptos Blockchain! Your efforts and responsible disclosure are appreciated, and every effort will be made to acknowledge your contributions in a timely manner.
+If you haven't done so already, please **enable two-factor auth** in your GitHub account.
+
+Send the email from an email domain that is less likely to get flagged for spam by gmail.
+
+This is an actively monitored account, the team will quickly respond.
+
+If you do not receive a response within 24 hours, please directly followup with the team in [Discord](https://discord.gg/aptoslabs). by reaching out to anyone with the role “Aptos Labs”.
+
+As above, please DO NOT include attachments or provide detail regarding the security issue in this email.
+
+## Incident Response Process
+
+1. Establish a new draft security advisory
+    1. In response to an email to [security@aptosfoundation.org](mailto:security@aptosfoundation.org), a member of the Aptos Labs will create a new draft security advisory for the incident at [https://github.com/aptos-labs/aptos-core/security/policy](https://github.com/aptos-labs/aptos-core/security/policy).
+    2. Add the reporter's GitHub account and relevant individuals to the draft security advisory.
+    3. Respond to the reporter by email, sharing a link to the draft security advisory.
+2. Responder should add appropriate content to the draft security advisory. To be eligible for a bug bounty, this includes:
+    1. A clear description of the issue and the impacted areas.
+    2. The code and the methodology to reproduce the underlying issue.
+    3. Discussion of potential remediations.
+3. Triage
+    1. Validate the issue.
+    2. Determine the criticality of the issue.
+    3. If this is not an issue but a bug, recommend to the submitter create an issue
+4. Resolve by building new docker containers that contain the fix and pushing to affected entities.
+
+## Bug Bounties
+
+Aptos Foundation offers bounties for security reports. Reports will be validated against the GitHub repository branch labeled "main" and no others. Moreover, any code within `main` but not in an actively deployed network or in a position to be deployed in such an environment, such as experimental or actively developed code, are out of scope for the bug bounty program. Production environments include the Aptos testnet and mainnet.
```

Comment:
This is kind of inconsistent. `main` is not deployed but `mainnet` is. Also from Move we do not take `main` but `aptos`. `main` as well as `aptos` can have bugs which we just introduced but they are not yet shipped.

I'm not sure what is the best way out of this. Perhaps as follows:

---
Reports will be validated against the GitHub repository branch labeled "mainnet" and no others. Moreover, any code within `mainnet` but not in an actively deployed network or in a position to be deployed in such an environment, such as experimental or actively developed code, are out of scope for the bug bounty program. *In addition, if a bug discovered in `mainnet` is already fixed in `main`, it is out of scope.* Production environments include the Aptos testnet and mainnet.